.travis.yml

```diff
@@ -0,0 +1,36 @@
+
+language: android
+sudo: required
+jdk: oraclejdk8
+
+android:
+  components:
+    - tools
+    - platform-tools
+    - build-tools-29.0.2
+    - android-29
+    - extra-google-google_play_services
+    - extra-google-m2repository
+    - extra-android-m2repository
+
+licenses:
+    - 'android-sdk-license-.+'
+    
+before_install:
+  - yes | sdkmanager "platforms;android-29"
+  - chmod +x ./gradlew
+
+script:
+  - echo "Hello! This is Travis"
+  - ./gradlew build
+
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+    - $HOME/.android/build-cache
+   
```

Comment:
@RaghavAwasthi Please parse yml file through [YAML Formatter](https://jsonformatter.org/yaml-formatter), the .travis.yml have basic indentation related syntax issues.

Reply:
Are you sure that there are syntax issues?
As I tested it on my Fork with Travis and it is successfully building as per requirements.
Build link:-https://travis-ci.com/github/RaghavAwasthi/barview-android/builds/154100914
Anyways I have parsed the yml file as you requested.

Reply:
Yes I was sure thats why I suggested you. In long run of the project the travis should be proper.